packages/nocodb-sdk/build-script/mergeAndGenerateSwagger.js

```
@@ -2,13 +2,25 @@ const {writeFileSync, readFileSync} = require('fs');
 const swaggerEE = JSON.parse(readFileSync('../nocodb/src/ee/schema/swagger.json', 'utf8'));
-const swaggerV3 = JSON.parse(readFileSync('../nocodb/src/schema/swagger-v3.json', 'utf8'));
+const swaggerV3 = JSON.parse(readFileSync('../nocodb/src/schema/swagger-v3.json', 'utf8'), (key, value) => {
+  if (key === '$ref') {
+    return value.replace(/^(#\/components\/schemas\/)(\w+)$/, '$1$2V3');
+  }
+  return value;
+});
+swaggerV3.components.schemas = Object.entries(swaggerV3.components.schemas).reduce((acc, [key, value]) => {
+  return {
+    [key + 'V3']: value,
+    ...acc
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)